arch/sim/src/sim/posix/sim_hostfs.c

```diff
@@ -279,7 +292,7 @@ int host_ioctl(int fd, int request, unsigned long arg)
 {
   /* Just call the ioctl routine */
 
-  return ioctl(fd, request, arg);
```

Comment:
need check the return value and get errno in failure case

Reply:
Done